### PR TITLE
[PORT] Long range gas analyzer

### PR DESCRIPTION
**Pull request by warface1234455**
- port https://github.com/tgstation/tgstation/pull/68051

This one is a useful to analyze things from far away
# Document the changes in your pull request
- Add a new long range gas analyzer (This can be printed from engi lathe and sci lathe after researching advanced eng tech)
- RD and CE will spawn with one
- One will spawn in CE and RD locker aswell

# Spriting
image

# Wiki Documentation
- Add a new long range gas analyzer
- RD and CE will spawn with one
- One will spawn in CE and RD locker aswell

# Changelog

:cl:  TheBoondock, warface1234455
rscadd: Add a new long range gas analyzer (This can be printed from engi lathe and sci lathe after researching advanced eng tech)
rscadd: RD and CE will spawn with one
rscadd: One will spawn in CE and RD locker aswell
/:cl:

### CODE REVIEW

**Comment by Simplehorror** [comment]
here comes the powercreep. before you know it, they'll be in the protofabs too. and then we'll get rid of the analyser and make it round-start equipment.

**Comment by ShoziOffical** [comment]
> here comes the powercreep. before you know it, they'll be in the protofabs too. and then we'll get rid of the analyser and make it round-start equipment.

what are you gonna powercreep with the analyzer lmao, like its here to scan gas and fuck anomalies

**Review by MajManatee — CHANGES_REQUESTED** [review]
Please make notes of ALL changes in the PR.

**Comment by ShoziOffical** [comment]
> Please make notes of ALL changes in the PR.

Which part is missing

File: code/modules/research/techweb/all_nodes.dm [hunk 1]
```diff
@@ -70,7 +70,7 @@
 	display_name = "Advanced Biotechnology"
 	description = "Advanced Biotechnology"
 	prereq_ids = list("biotech")
-	design_ids = list("hypospray", "piercesyringe", "hypospraypierceupg", "hyposprayspeedupg", "pinpointer_crew", "smoke_machine", "plasmarefiller", "limbgrower", "meta_beaker", "healthanalyzer_advanced", "harvester", "holobarrier_med", "detective_scanner", "detective_scanner_advanced" , "defibrillator_compact", "vialbox")
+	design_ids = list("hypospray", "piercesyringe", "hypospraypierceupg", "hyposprayspeedupg", "pinpointer_crew", "smoke_machine", "plasmarefiller", "limbgrower", "meta_beaker", "healthanalyzer_advanced", "harvester", "holobarrier_med", "detective_scanner", "detective_scanner_advanced" , "defibrillator_compact", "vialbox", "rangedanalyzer")
```
Comment (MajManatee) [inline code/modules/research/techweb/all_nodes.dm]: Why would it be here and not under advanced engineering?

**Comment by MajManatee** [comment]
the reduction of ALL scanner cooldowns to 1/5th of the original speed

**Comment by MajManatee** [comment]
the cooldown reduction

File: code/modules/research/techweb/all_nodes.dm (hunk 1 above)
Comment (ShoziOffical) [reply code/modules/research/techweb/all_nodes.dm]: What

Comment (ShoziOffical) [reply code/modules/research/techweb/all_nodes.dm]: Fuck my eyes

**Comment by ShoziOffical** [comment]
> the cooldown reduction

I don't think this needs a cooldown since it's only for gas scanning

**Comment by MajManatee** [comment]
Thats fine, just add it to the description

**Review by Moltijoe — APPROVED** [review]
Lemme just walk into this plasmaflood to check its temperature and gas composition

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**